src/promptflow/promptflow/_cli/_pf/_flow.py

```diff
@@ -270,7 +282,46 @@ def _init_existing_flow(flow_name, entry=None, function=None, prompt_params: dic
     print(f"Done. Generated flow in folder: {flow_path.resolve()}.")
 
 
-def _init_flow_by_template(flow_name, flow_type, overwrite=False):
+def _init_chat_flow(flow_name, flow_path, connection=None, deployment=None):
+    from promptflow._sdk._configuration import Configuration
+
+    example_flow_path = Path(__file__).parent.parent / "data" / "chat_flow" / "flow_files"
+    for item in list(example_flow_path.iterdir()):
+        _copy_to_flow(flow_path=flow_path, source_file=item)
+
+    # Generate flow.dag.yaml to chat flow.
+    connection = connection or DEFAULT_CONNECTION
+    deployment = deployment or DEFAULT_DEPLOYMENT
+    ChatFlowDAGGenerator(connection=connection, deployment=deployment).generate_to_file(flow_path / "flow.dag.yaml")
+    # When customer not configure the remote connection provider, create connection yaml to chat flow.
+    is_local_connection = Configuration.get_instance().get_connection_provider() == ConnectionProvider.LOCAL
+    if is_local_connection:
+        OpenAIConnectionGenerator(connection=connection).generate_to_file(flow_path / "openai.yaml")
```

Comment:
Nit: add a log when we are creating or not creating connection file?

Reply:
It will log when create connection files.
![image](https://github.com/microsoft/promptflow/assets/17938940/b46b0dab-e712-411b-a9f3-8d247ef31555)